public/js/scripts.min.js

```diff
@@ -1 +1 @@
.033-0.667-0.117-1.8-0.25c-0.9-0.133-1.683-0.283-2.35-0.45c-2.066-0.533-3.783-1.5-5.15-2.9c-1.033-1.067-1.9-2.783-2.6-5.15C1.317,48.867,1.133,48.117,1,47.35L0.8,46.4c-0.133-1.133-0.267-2.55-0.4-4.25C0.133,38.717,0,35.583,0,32.75V26.4c0-2.833,0.133-5.95,0.4-9.35l0.4-4.25c0.167-0.966,0.417-2.05,0.75-3.25c0.7-2.333,1.567-4.033,2.6-5.1c1.367-1.434,2.967-2.434,4.8-3c0.633-0.167,1.333-0.3,2.1-0.4c0.4-0.066,0.917-0.133,1.55-0.2c4.9-0.333,11.283-0.567,19.15-0.7C35.65,0.05,39.083,0,42.05,0L45,0.05c2.467,0,4.933,0.034,7.4,0.1c7.833,0.133,14.2,0.367,19.1,0.7c0.3,0.033,0.833,0.1,1.6,0.2c0.733,0.1,1.417,0.233,2.05,0.4c1.833,0.566,3.434,1.566,4.8,3c1.066,1.066,1.933,2.767,2.6,5.1c0.367,1.2,0.617,2.284,0.75,3.25l0.4,4.25C84,20.45,84.15,23.567,84.15,26.4z M33.3,41.4L56,29.6L33.3,17.75V41.4z"></path>'),c.push('<polygon fill-rule="evenodd" clip-rule="evenodd" fill="#FFFFFF" points="33.3,41.4 33.3,17.75 56,29.6"></polygon>'),c.push("</svg>"),c.push("</div>"),c.push("</div>"),c.push('<div class="html5-info-bar">'),c.push('<div class="html5-title">'),c.push('<div class="html5-title-text-wrapper">'),c.push('<a id="lazyYT-title-',u,'" class="html5-title-text" target="_blank" tabindex="3100" href="//www.youtube.com/watch?v=',u,'">'),c.push(r),c.push("</a>"),c.push("</div>"),c.push("</div>"),c.push("</div>"),a.css({"padding-bottom":l}).html(c.join("")),i=s>640?"maxresdefault.jpg":s>480?"sddefault.jpg":s>320?"hqdefault.jpg":s>120?"mqdefault.jpg":0==s?"hqdefault.jpg":"default.jpg",n=a.find(".ytp-thumbnail").css({"background-image":["url(//img.youtube.com/vi/",u,"/",i,")"].join("")}).addClass("lazyYT-image-loaded").on("click",function(t){t.preventDefault(),!a.hasClass("lazyYT-video-loaded")&&n.hasClass("lazyYT-image-loaded")&&a.html('<iframe src="//www.youtube.com/embed/'+u+"?autoplay=1&"+h+'" frameborder="0" allowfullscreen></iframe>').addClass("lazyYT-video-loaded")}),t.getJSON("//gdata.youtube.com/feeds/api/videos/"+u+"?v=2&alt=json",function(t){a.find("#lazyYT-title-"+u).text(t.entry.title.$t)})}t.fn.lazyYT=function(e){var l={loading_text:"Loading...",default_ratio:"16:9",callback:null,container_class:"lazyYT-container"},n=t.extend(l,e);return this.each(function(){a(t(this).addClass(n.container_class),n)})}}(jQuery),$(document).ready(function(){$(".lazyload-youtube").lazyYT(),$(".nav-button").click(function(){$("nav").toggleClass("nav--shown"),$("nav").hasClass("nav--shown")?$(".nav-button").text("Close"):$(".nav-button").text("Menu")}),$(".ask-confirmation").click(function(t){t.preventDefault(),confirm("Are you sure you wish to perform this action?")&&(window.location.href=$(this).attr("href"))}),$(".attending-button").click(function(t){var a=$(this);t.preventDefault(),$.post(a.attr("href")).done(function(t){if(t.attending)return a.addClass("is-attending"),void a.text(t.isPast?"I went to this":"I will be there");a.removeClass("is-attending"),a.text(t.isPast?"I wasn't there":"I won't be there")}).fail(function(){alert("Could not toggle your attendance status...")})})});
```

Comment:
Why has this changed?

Reply:
@jezemery @lboynton maybe something didn't get changed before? Jez could you try generating this again just to be sure please? If it's really changed that's ok :+1:

Reply:
Looking at the diff the two versions are logically the same, so must be minifier difference..? Probably fine anyway.

Reply:
It was the minifier, I assumed there was perhaps some legacy versioning or something that changed